aiida/engine/processes/futures.py

```diff
@@ -59,7 +59,12 @@ def __init__(
 
             # Try setting up a filtered broadcast subscriber
             if self._communicator is not None:
-                broadcast_filter = kiwipy.BroadcastFilter(lambda *args, **kwargs: self.set_result(node), sender=pk)
+
```

Comment:
Thanks - I was going to suggest to add a comment here as to why that is necessary but then I realized that this is in the `ProcessFuture` 

https://github.com/aiidateam/aiida-core/issues/4758

Reply:
yeh I thought that might be the case